prelude/prelude.purs

```diff
@@ -74,14 +84,23 @@ module Prelude
   infixr 0 $
   infixl 0 #
 
+  -- | Applies the first argument to the second, making a backward pipe.
+  -- |
+  -- | `reverse $ "str" ++ "ing" == "gnirts"`
   ($) :: forall a b. (a -> b) -> a -> b
   ($) f x = f x
 
+  -- | Applies the second argument to the first, making a forwards pipe.
+  -- |
+  -- | `"str" ++ "ing" # reverse == "gnirts"`
   (#) :: forall a b. a -> (a -> b) -> b
   (#) x f = f x
 
   infixr 6 :
 
+  -- | Prepends an element to a list.
+  -- |
+  -- | `"try" : ["purescript", "today"] == ["try", "purescript", "today"]`
```

Comment:
In the future, this may be generalised so that it also applies to other data structures, with different complexity. Do we really want to add that kind of info to the documentation?